src/TestFramework/Coverage/XmlReport/JUnitTestFileDataProvider.php

```diff
@@ -120,7 +121,9 @@ private static function createXPath(string $jUnitPath): SafeDOMXPath
         }
 
         $dom = new DOMDocument();
-        $dom->load($jUnitPath);
+        $success = @$dom->load($jUnitPath);
+
+        Assert::true($success);
```

Comment:
is there any errors we should up here in case of failure? (same in the other places)

Reply:
Normally we could expect valid coverage reports, so this is just for a case where we hit something very rare. So I don't think there's enough evidence to warrant a specific error type.